### PR TITLE
Show measurement

### LINKED ISSUE #45: Results: See measurement (webview)

### PR DESCRIPTION
**Pull request by sdsantos**
Closes #45

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -52,6 +52,9 @@ sqldelight-native = { module = "app.cash.sqldelight:native-driver", version.ref
 # Files
 okio = { module = "com.squareup.okio:okio", version = "3.9.0" }
 
+# WebView
+webview = { module = "io.github.kevinnzou:compose-webview-multiplatform", version = "1.9.20" }
+
```
Comment (aanorbel) [inline gradle/libs.versions.toml:57]: Do we want to use a webview or launch in a browser.

**Review by aanorbel — CHANGES_REQUESTED** [review]
I dont particularly agree with adding a webview if opening on device browser is just as effective.

File: composeApp/src/commonMain/kotlin/org/ooni/probe/domain/DownloadUrls.kt [hunk 2]
```diff
@@ -14,7 +14,7 @@ class DownloadUrls(
     suspend operator fun invoke(taskOrigin: TaskOrigin): Result<List<UrlModel>, Engine.MkException> =
         engineCheckIn(taskOrigin)
             .map { results ->
-                val urls = results.urls.map { it.toModel() }
+                val urls = results.urls.map { it.toModel() }.take(5)
```
[the remark this answers is not in the document]
Comment (sdsantos) [reply composeApp/src/commonMain/kotlin/org/ooni/probe/domain/DownloadUrls.kt:17]: Just some code I added for quicker tests, and forgot to remove.

**Review by aanorbel — APPROVED** [review]
🐳

**Pull request merged**